packages/contracts-bedrock/test/governance/ProposalValidator.t.sol

```diff
@@ -112,7 +112,7 @@ contract ProposalValidator_Init is CommonTest {
     function _getProposalTypesRequiredApprovals()
         internal
         pure
-        returns (ProposalValidator.ProposalType[] memory, uint256[] memory)
+        returns (ProposalValidator.ProposalType[] memory, ProposalValidator.ProposalTypeData[] memory)
```

Comment:
you return the whole struct here however the name of the function only suggests it returns the required approvals.